modules/client/src/addresslistVoting/internal/utils.ts

```diff
@@ -99,8 +105,9 @@ export function toAddresslistVotingProposalListItem(
     parseInt(proposal.startDate) * 1000,
   );
   const endDate = new Date(parseInt(proposal.endDate) * 1000);
+  const decodedProposalId = decodeProposalIdSubgraph(proposal.id);
   return {
-    id: proposal.id,
+    id: encodeProposalId(decodedProposalId.pluginAddress, decodedProposalId.id),
```

Comment:
Why not simply passing `id: proposal.id`? Instead of decoding and reencoding

Reply:
was necessary because of the different formats. this is now removed because of https://github.com/aragon/sdk/pull/157#discussion_r1111935932